mod/caching/set/type_plus_right/wikirate_analysis/metric.rb

```diff
@@ -11,14 +11,15 @@
 
 # update if topic taging is touched
 ensure_set { TypePlusRight::Metric::WikirateTopic }
-expired_cached_count_cards(
-  set: TypePlusRight::Metric::WikirateTopic
-) do |changed_card|
-  card_names = changed_card.item_names.unshift('in')
-  Card.search type_id: Card::WikirateAnalysisID,
-              right: card_names,
-              append: 'metric'
-end
+
+# expired_cached_count_cards(
+#   set: TypePlusRight::Metric::WikirateTopic
+# ) do |changed_card|
+#   card_names = changed_card.item_names.unshift('in')
+#   Card.search type_id: Card::WikirateAnalysisID,
+#               right: card_names,
+#               append: 'metric'
+# end
```

Comment:
So, I'm hoping the above is unnecessary, because it takes metric creation from a couple of seconds to a couple of minutes.